src/stylesheets/components/_tabs.scss

```diff
@@ -98,10 +98,26 @@
   padding-bottom: $govuk-spacing-scale-6;
 }
 
+.app-c-tabs__copy {
```

Comment:
I totally assumed this was relating to the copy button because of the class name. Can we change it to… err…
 something else?